platform.md

```diff
@@ -32,13 +32,13 @@ Examples of a platform might include:
       - [Rebase](#rebase)
       - [Launch](#launch)
     - [Usage](#usage)
-      - [`detector`](#detector)
-        - [Inputs](#inputs)
-        - [Outputs](#outputs)
       - [`analyzer`](#analyzer)
         - [Inputs](#inputs-1)
         - [Outputs](#outputs-1)
         - [Layer analysis](#layer-analysis)
+      - [`detector`](#detector)
+        - [Inputs](#inputs)
+        - [Outputs](#outputs)
```

Comment:
You can use the VScode plugin mentioned in the [comment](https://github.com/buildpacks/spec/pull/197/files#diff-e603760990971da3f77be4bb8d77c3405098f006814fd8c054d2d15f395b8330R15) above the TOC to regenerate consistent anchor links (e.g. `#outputs-2`).